lib/logstash/inputs/file.rb

```diff
@@ -119,6 +135,11 @@ class LogStash::Inputs::File < LogStash::Inputs::Base
   # set the new line delimiter, defaults to "\n"
   config :delimiter, :validate => :string, :default => "\n"
 
+    # set the mode, defaults to "tail"
+  # in read_to_eof mode filewatch will signal the EOF with a "\x04" control character
+  # in tail mode behaviour is as before
+  config :ignore_after, :validate => :number, :default => 24 * 60 * 60
```

Comment:
sure, the comment needs to change.
but I spoke to Tudor and they are having real issues with the name and the two functions controlled by one option thing - I asked him to comment on our issue, he did: https://github.com/logstash-plugins/logstash-input-file/issues/81#issuecomment-165085337